ibm/service/vpc/data_source_ibm_is_instances.go

```diff
@@ -121,6 +121,32 @@ func DataSourceIBMISInstances() *schema.Resource {
 							Computed:    true,
 							Description: "Indicates whether the metadata service endpoint is available to the virtual server instance",
 						},
+						isInstanceMetadataService: {
+							Type:        schema.TypeList,
```

Comment:
As of now the values are not getting populated in any of the data sources, Please review the changes again